loky/backend/_posix_reduction.py

```diff
@@ -11,13 +11,10 @@
 import socket
 import _socket
 
-from .reduction import register
-from .context import get_spawning_popen
+from multiprocessing.connection import Connection
 
-if sys.version_info >= (3, 3):
-    from multiprocessing.connection import Connection
-else:
-    from _multiprocessing import Connection
+from .reduction import _dispatch_table
```

Comment:
Let's implement a public function called `register` in `.reduction` and import that instead of `_distpach_table`.